File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int index=0;
        int pre=nums[0];
        while(index==0){
            nums[index]=pre;
            index=index+1;
        }
        for(int i=1;i<nums.length;i++)
        {
            if(pre!=nums[i])
            {
                nums[index++]=nums[i];
                pre=nums[i];
            }
        }
        return index;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int>st;
        for(int i=0;i<=nums.size()-1;i++){
            st.insert(nums[i]);
        }
        int j=0;
        int k=st.size();
        for(int x:st){
            nums[j++]=x;
        }
        return k;
    }
};